tensorflow/c/c_api.cc

```diff
@@ -2803,4 +2803,40 @@ TF_Buffer* TF_GetRegisteredKernelsForOp(const char* name, TF_Status* status) {
   }
   return ret;
 }
+
+// TF_Server functions ----------------------------------------------
```

Comment:
You may want to hide all of this inside an `#ifdef __ANDROID__` similar to https://github.com/tensorflow/tensorflow/blob/1737b809b7a7b258c81a77dd4e4670315c70b53d/tensorflow/c/c_api.cc#L2249 , since we don't include the required dependencies when building for Android (as per the BUILD file changes), which is probably a good thing since there doesn't seem to be a need for this in Android builds yet.

(So, these functions should fail if invoked in Android)

Reply:
Done, wrapped the whole block.